oxidation/crates/parsec_core_fs/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+mod error;
+#[allow(dead_code)]
+mod extensions;
+mod remote_loader;
+mod schema;
+mod storage;
+mod workspacefs;
+
+#[macro_use]
+extern crate diesel;
+
+pub use error::*;
+pub use storage::*;
+
+pub enum Language {
```

Comment:
`libparsec` should be separated to the GUI, so you should not handle language in the Rust code (besides error messages retuned by liparsec are technical one, so no need to translate them)

Reply:
This enum is used in FileTransactions

Reply:
Oh yes, this to know how to rename a file conflict: `foo.txt` -> `foo (conflict with Bar).txt` in english and `foo (conflit avec Bar).txt` in french

I don't like this stuff (because it brings translation issues in an unrelated codebase), but @vxgmichel said it's a killer feature so I guess it's here to stay 😄

Reply:
You should add a `// Translations used to for File/Folder conflict renaming` comment